src/test/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasSlicingTest.java

```diff
@@ -1030,11 +1030,13 @@ public void testMultiPolygonWithOverlappingSlicedInners()
     public void testMultiPolygonWithOverlappingUnslicedInners()
     {
         final Atlas rawAtlas = this.setup.getMultiPolygonWithOverlappingUnslicedInners();
+        System.out.println(rawAtlas.toStringDetailed());
```

Comment:
think this was left included by accident!

Reply:
Ooooh good catch!